qcut/scripts/e2e-virtual-display-win.ps1

```
@@ -0,0 +1,114 @@
+            # The desktop index tells us which desktop to close
+            $targetIndex = [int]$DesktopId
+
+            # Switch to the target desktop
+            # Win+Ctrl+Right arrow navigates between desktops
+            $currentCount = Get-DesktopCount
+
+            if ($currentCount -gt 1) {
+                # Close current desktop (assumes we're on the test desktop)
+                Send-Keys "^#{F4}"
+                Start-Sleep -Milliseconds 500
+                Write-Host "Virtual desktop removed"
```

PSScriptAnalyzer (1.24.0)

[warning] File 'e2e-virtual-display-win.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)